website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -544,7 +545,8 @@ default setting set during ORCA installation. For more information, see the docu
   "meta": {
     "granuleRecoveryWorkflow": "OrcaRecoveryWorkflow",
     "excludeFileTypes": [".cmr", ".xml", ".met"],
-    "multipart_chunksize_mb": 400
+    "multipart_chunksize_mb": 400,
+    "orcaDefaultBucketOverride": "glacier"
```

Comment:
Instead of `"glacier"` for the value maybe have this as a more realistic bucket name like `"prod_orca_worm"` or something like that anyway. I would really like for us to detach from the word `glacier` to something more generic since in AWS there are essentially 3 glacier storage classes now and if we ever need to go to Azure or Google cloud service storage it would be good to be somewhat generic in our approach so that we are not telegraphing the underneath to users.

Reply:
Sounds good. Updated.